### PR TITLE
Feat: Unify Storybook Documentation

### PR DESCRIPTION
**Pull request by renatamottam**
Cherry-pick from Gatsby project: https://github.com/vtex-sites/gatsby.store/pull/26

## What's the purpose of this pull request?
This PR focused on unify and improve the Theming documentation on Storybook. For that we:
- Added base files (`Getting Started`, `Theming`, `Colors`, `Typography` and `Icons`)
- Updated the naming of tokens
- Defined an initial structure to list the components in the Storybook
- Listed 2 components as an example: `BannerText` and `Toggle`

## How does it work?
We're proposing a new way to use Storybook files as `docs` only. 
Also, we updated tokens naming scheme:
image

image

image

## How to test it?
Local: `yarn storybook --docs`
Remote: [Preview Link](https://gatsby-store-storybook-git-feat-fsss-319-story-0afc09-faststore.vercel.app/)

## References
- [MDX Stories documentation](https://storybook.js.org/docs/react/writing-docs/mdx)

## Checklist

<em>You may erase this after checking them all ;)</em>

- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#12](https://github.com/vtex-sites/gatsby.store/pull/12))* 
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`

### CODE REVIEW

**Review by hellofanny — APPROVED** [review]
I checked it on the [preview](https://vercel.com/faststore/nextjs-store-storybook/9cJiBWqZaEosi1eZLQvSCHCgahsG) link it seems fine! LGTM! 🚀 🚀 🚀

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Added base files (`Getting Started`, `Theming`, `Colors`, `Typography` and `Icons`) to Storybook ([#26](https://github.com/vtex-sites/nextjs.store/pull/67))
```
Comment (eduardoformiga) [inline CHANGELOG.md:26]:
```suggestion
- Added base files (`Getting Started`, `Theming`, `Colors`, `Typography` and `Icons`) to Storybook ([#67](https://github.com/vtex-sites/nextjs.store/pull/67))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Updated tokens' naming scheme ([#26](https://github.com/vtex-sites/nextjs.store/pull/67))
```
Comment (eduardoformiga) [inline CHANGELOG.md:27]:
```suggestion
- Updated tokens' naming scheme ([#67](https://github.com/vtex-sites/nextjs.store/pull/67))
```

File: src/stories/getting-started.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+<Meta title="Getting Started" />
+
+<header>
+
+# Getting Started
+
+</header>
+
+FastStore gatsby.store is an ecommerce-focused library that provides best-of-breed components, blazing-fast performance, and a frictionless development experience to create storefronts using [FastStore UI library](https://www.faststore.dev/reference/ui/faststore-ui) as base.
```
Comment (eduardoformiga) [inline src/stories/getting-started.stories.mdx:12]:
```suggestion
FastStore nextjs.store is an ecommerce-focused library that provides best-of-breed components, blazing-fast performance, and a frictionless development experience to create storefronts using [FastStore UI library](https://www.faststore.dev/reference/ui/faststore-ui) as base.
```

**Review by eduardoformiga — APPROVED** [review]
Amazing job! ❤️ 

I left some little comments.

File: src/stories/getting-started.stories.mdx (hunk 3 above)
Comment (hellofanny) [reply src/stories/getting-started.stories.mdx:12]: very well noticed hahaha 💯

**Review by lucasfp13 — APPROVED** [review]
Checked through Storybook, LGTM!

**Pull request merged**